modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -34,6 +34,26 @@ include::hello-world:page$start-using-sdk.adoc[tag=install]
 Python SDK 3.n automatically bundles libcouchbase (downloading and building it if necessary),
  so no need to install it separately.
 
+== Version 3.0.4 (22 July 2020)
+
+[source,bash]
+----
+pip install couchbase==3.0.4
```

Comment:
Did you still want to change to 
python3 -m pip install couchbase==3.0.4
or just in the install docs?